packages/adapter-cloudflare/index.js

```diff
@@ -217,6 +221,21 @@ function get_routes_json(builder, assets, { include = ['/*'], exclude = ['<all>'
 				return builder.prerendered.paths;
 			}
 
+			if (rule === '<redirects>' && existsSync(redirects_file)) {
+				const redirects = readFileSync(redirects_file, 'utf8')
+					.split('\n')
+					.filter(Boolean)
```

Comment:
might need to filter out lines that are just white space to prevent it from throwing an error on those?
```suggestion
					.map((s) => s.trim())
					.filter(Boolean)
```